samples/secret/secret.yaml

```diff
@@ -53,6 +53,13 @@ isilonClusters:
     # Examples: "0777", "777", "0755"
     # isiVolumePathPermissions: "0777"
 
+    # Unique ID if the certificate used to encrypt replication policy
```

Comment:
Add is before used.
Unique ID if the certificate is used to encrypt replication policy

Reply:
done